release-tools/metabuild.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+D=$1
+MAKEBSD=$2
+CFLAGS_HARDCODED=$3
+
+OUT=Makefile
+
+# Set up directories
+cd ${D}
+SUBDIR_DEPTH=`${MAKEBSD} -V SUBDIR_DEPTH`
+
+# Set up *-config.h so that we don't have missing headers
+rm -f ${SUBDIR_DEPTH}/cpusupport-config.h
+touch ${SUBDIR_DEPTH}/cpusupport-config.h
+
+copyvar() {
+	var=$1
+	printf "$var=$(${MAKEBSD} -V $var)\n" >> $OUT
```

Comment:
I would spell this as

```
printf "%s=" "$var" >> $OUT
${MAKEBSD} -V "$var" >> $OUT
```

to avoid ugly subshells (and also the risk of multiple-shell-expansion issues, although those are *hopefully* not likely to arise in this context).

Reply:
fixed in a REBASE commit.